sdk/core/Azure.Core.TestFramework/src/RecordedTestAttribute.cs

```diff
@@ -26,20 +26,30 @@ public TestCommand Wrap(TestCommand command)
             {
                 test = test.Parent;
             }
-            if (test.Fixture is RecordedTestBase fixture && fixture.Mode == RecordedTestMode.Playback)
+
+            if (test.Fixture is RecordedTestBase fixture)
             {
-                return new FallbackCommand(command);
+                return fixture.Mode == RecordedTestMode.Playback ? new PlaybackCommand(command) : new RecordedTestAttributeCommand(command);
```

Comment:
This naming is a bit confusing. What do you think about something like:
```suggestion
                return fixture.Mode == RecordedTestMode.Playback ? new AutoReRecordingCommand(command) : new IgnoreServiceErrorsCommand(command);
```

I realize this may not make sense in the future if the capabilities of the IgnoreServiceErrorsCommand expands its functionality, but for now it makes things very clear.

Reply:
As for PlaybackCommand, I think we would only have at most 1 command per mode.

Reply:
What about RecordedTestAttributePlaybackCommand and RecordedTestAttributeCommand?

Reply:
I was just thinking that commands should be discrete and named for their functionality rather than be defined by the recording mode they apply to.

Reply:
I think that happens to be true today, but not because it must be true of all command functionality.

Reply:
Totally agree - in the future it may be the case that this proposed strategy of using separate modes per command won't actually work, and we would need to re-think it at that time. But my hope is that we would add additional playback functionality into the Playback command.